README.md

```diff
@@ -44,7 +45,7 @@ You can download and install a PowerShell package for any of the following platf
 | macOS 10.12                        | [.pkg][rl-macos]       | [Instructions][in-macos]      |
 | Docker                             |                        | [Instructions][in-docker]     |
 | Kali Linux                         | [.deb][rl-ubuntu16]    | [Instructions][in-kali]       |
-| Raspbian (Stretch)                 | [.tgz][rl-raspbian]    | [Instructions][in-raspbian]   |
+| Raspbian (Stretch) *Experimental*  | [.tgz][rl-raspbian]    | [Instructions][in-raspbian]   |
```

Comment:
Travis updated the readme.md to include `win-arm32` and `win-arm64` zip packages in #5688. Can you please review that and give your feedback?